.github/dockerfiles/ov_test/debian_10_arm/Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG REGISTRY="docker.io"
+FROM ${REGISTRY}/library/debian:10.13
+
+USER root
+
+# APT configuration
+RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
+    echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::Fix-Broken "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    TZ="Europe/London"
+
+RUN apt-get update && \
+    apt-get install  \
+        git \
+        libc6-dev \
+        # Python for running C++ Functional tests via parallelization script  \
```

Comment:
is this only one difference with build image?

Reply:
No. The build image also contains building Python from the source and installing `sccache`.